replica/backup.go

```diff
@@ -238,6 +238,7 @@ func CreateHoles() error {
 			time.Sleep(1)
 			retryCount++
 			if retryCount == 5 {
+				_ = s.Close()
```

Comment:
this is NOT safe to do as other go routines still have fd and doing IOs on it

Reply:
This is safe to do @vishnuitta since s.Close will acquire lock and then close the fds. So I/O's will  get blocked and when lock is free those goroutine will check if s.r  will be set to nil.